db/migrate/20240529093425_make_republishing_events_bulk_false_by_default.rb

```diff
@@ -0,0 +1,7 @@
+class MakeRepublishingEventsBulkFalseByDefault < ActiveRecord::Migration[7.1]
+  def up
+    change_column :republishing_events, :bulk, :boolean, default: false
+  end
```

Comment:
I thought it was previously only defaulting to `false` for existing records, but new records were correctly erroring if no value was provided? Will this mean we could end up with an accidental default in future?

Reply:
You're right - I forgot we were passing in the `bulk: false` when Republishing. Will update the commit message. As we need a default, I think `false` is a sensible one, and safe.

Reply:
Or are you suggesting we should ignore the error instead?

Reply:
As discussed, we're now migrating all existing `RepublishingEvent`s `bulk` values where `nil` to `false`, then unset the default value to prevent accidentally creating non bulk `RepublishingEvent`s